assets/src/tinymce/__tests__/components/Modal.test.js

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
The directory should be renamed tests from __tests__

Reply:
@spacedmonkey This was discussed before [here](https://github.com/rtCamp/web-stories-wp/pull/194#discussion_r565848120)

Reply:
Fixed.